### ISSUE TITLE
Possible to hide search and assist icons in toolbar?

### ISSUE THREAD

**Issue by rdeneer** [j-a-n/lovelace-wallpanel#127]
Many thanks for developing wallpanel, very happy with this mod for my wall mounted tablet!

I'd like to show the upper panel toolbar so I can use subviews (which saves me the hassle of adding back buttons manually).
However, this implies that the search and assist buttons are visible on the main view:
![buttons2](https://github.com/j-a-n/lovelace-wallpanel/assets/17760057/d0690a6a-dc1d-4de2-830e-de067270a50b)
Users can access any entity through these buttons but unfortunately HA does not allow the user to disable these. Would it be possible to add a feature to wallpanel to hide/show these buttons?

I know some users have achieved this through card mod:

```
  # Header
  card-mod-theme: remove-the-fucking-bitchass-icons
  card-mod-root-yaml: |
    paper-tabs$: |
      /* This hides tab scroll arrows */
      paper-icon-button {
        display: none !important;
      }
    ha-app-layout$: |
        /* This makes sure the bottom scrollbar isn't hiding cards */
        #contentContainer {
          padding-top: 0px !important;
          padding-bottom: 48px !important;
        }
    .: |
      .action-items {
        display: none !important;
      }
```

**Comment by j-a-n** [comment]
Just added this feature in version 4.13.0.
Just set `hide_toolbar_action_icons` to `true`.

**Comment by rdeneer** [comment]
Many thanks! Just tested this and the only issue I'm having is that the three dots next to the assist icon is also hidden. 
![Schermafbeelding 2023-07-10 163433](https://github.com/j-a-n/lovelace-wallpanel/assets/17760057/ec9e50d9-d73d-4830-a136-500bbfbbb7b7)
This makes it impossible to access the "Configure UI" option for editing. Would it be possible to ignore the `hide_toolbar_action_icons` when `wp_enabled = false`? This way the three dots are shown when `wp_enabled = false`.

**Comment by kbrown01** [comment]
Exactly!!!! I actually like sidebars and have separate dashboards for several things (all others are hidden in the UI for my pads).
I was testing and setup a sample dashboard (lucky it was a sample I think).
I got things the way I wanted and hid the top toolbar.

This made it 100% impossible (to me) to edit the dashboard anymore. I could do nothing. 

Maybe someone can explain how to do it is top bar is hidden, I could not find it and had to delete the dashboard and start over with top bar shown.

**Comment by j-a-n** [comment]
Thanks @rdeneer , this is now fixed in v4.15.0
